index.js

```diff
@@ -1,27 +1,33 @@
-var fs = require('fs')
-var ncp = require('ncp')
+var fs = require('fs-extra')
 var path = require('path')
 var mkdirp = require('mkdirp')
 
 module.exports = {}
 
+module.exports.FALLBACK_DIR_NAME = 'MAPEO_PRESETS_FALLBACK_DIR'
+
 module.exports.unpack = function (root, cb) {
-  var presetRoot = path.join(root, 'presets')
+  // We need this because mapeo-default-settings does not have
+  // a subdirectory, but styles in this repo does have a subdirectory for
+  // each default style.
+  var presetRoot = path.join(root, 'presets', module.exports.FALLBACK_DIR_NAME)
   var styleRoot = path.join(root, 'styles')
   mkdirp(presetRoot, function (err) {
     if (err) return cb(err)
     mkdirp(styleRoot, function (err) {
       if (err) return cb(err)
-      ncp(path.join(__dirname, 'node_modules', 'mapeo-default-settings', 'build'), presetRoot, function (err) {
+      fs.copy(path.join(__dirname, 'node_modules', 'mapeo-default-settings', 'dist'), presetRoot, function (err) {
```

Comment:
Without 'dist', mapeo-desktop won't get the icons and iD will look baren and sad :(

But mapeo mobile uses the `build` directory. What's the difference?

Reply:
This was a breaking change I recently pushed in v3 of mapeo-default-settings, because we generally use the build folder for creating builds manually, and it can often have different artifacts in there which could accidentally get published. I had been using `dist` for the WCMC config, so I thought it would make more sense to use `dist` for the npm package. I haven't switched mobile yet.